packages/cli/src/lib/config.ts

```diff
@@ -44,10 +46,22 @@ async function createConfig(prompt: Prompt = new InquirerPrompt()): Promise<Conf
     const jdkInstaller = new JdkInstaller(process);
     jdkPath = await jdkInstaller.install(DEFAULT_JDK_FOLDER);
   }
-  const androidSdkPath = await prompt.promptInput('Path to the Android SDK:', null,
-      AndroidSdkTools.validatePath);
 
-  return new Config(jdkPath, androidSdkPath);
+  const sdkInstallRequest = await prompt.promptConfirm('Do you want Bubblewrap to install ' +
```

Comment:
@andreban I just noticed this with the code above as well - shouldn't these be in the strings file?

Reply:
Yes.

@joycetoh8, we're are adding all user-facing strings to `strings.ts`. This makes the code a bit cleaner and should help with internationalisation in the future.

Reply:
Moved strings into strings.ts. Thanks for the heads up!